python/adjoint/__init__.py

```diff
@@ -15,3 +15,8 @@
 from .filter_source import FilteredSource
 
 from .filters import *
+
+try:
+  from . import jax
+except ImportError as error:
+  print('Unable to import JAX.')
```

Comment:
Probably no output on error.  They should just get an exception if they try to use the JAX features.

Reply:
Sounds good. Removed the print statement.